AutomatedTesting/Gem/PythonTests/scripting/scripting_utils/scripting_constants.py

```diff
@@ -14,27 +14,38 @@
 SCRIPT_CANVAS_UI = "Script Canvas"
 NODE_PALETTE_UI = "Node Palette"
 ASSET_EDITOR_UI = "Asset Editor"
+SCRIPT_EVENT_UI = "Script Events"
 VARIABLE_MANAGER_UI = "Variable Manager"
 
 """
 String constants for Node Palette
 """
 NODE_CATEGORY_MATH = "Math"
 NODE_STRING_TO_NUMBER = "String To Number"
+NODE_TEST_METHOD = "test_method_name"
+
+"""
+Constants for Asset Editor
+"""
+DEFAULT_SCRIPT_EVENT = "EventName"
+DEFAULT_METHOD_NAME = "MethodName"
 
 """
 Constants for QtWidgets.
 Different from window/tab names because they do not have spaces
 """
 NODE_PALETTE_QT = "NodePalette"
 TREE_VIEW_QT = "treeView"
+EVENTS_QT = "Events"
 VARIABLE_PALETTE_QT = "variablePalette"
 VARIABLE_MANAGER_QT = "VariableManager"
 GRAPH_VARIABLES_QT = "graphVariables"
-ADD_BUTTON_QT = "addButton"
```

Comment:
Was this intentionally removed? I remember seeing this utilized in another test you've worked on.

Reply:
I did not intend to remove this. I think I accidentally clobbered it when I added more constants to the file